language/move-analyzer/src/symbols.rs

```diff
@@ -147,6 +147,8 @@ pub struct UseDef {
     def_loc: DefLoc,
     /// Location of the type definition
     type_def_loc: Option<DefLoc>,
+    /// Doc string for the relevant identifier/function
```

Comment:
I also realized that it might be more intuitive to make the `doc_string` part of `DefLoc` (possibly renaming it to `DefInfo`), particularly keeping in mind that we may want to optimize this code at some point and use a reference to `DefLoc` in `UseDef` rather than "inlining" it.

Reply:
Do you want this change in this PR? 

It will cause a lot of diffs across `symbols.rs` and it will require a few changes to the code itself. I don't think I can include a String inside `DefLoc` as it has the `Copy` trait and `DefLoc` is copied in multiple places across this file.

Reply:
> It will cause a lot of diffs across `symbols.rs` and it will require a few changes to the code itself. I don't think I can include a String inside `DefLoc` as it has the `Copy` trait and `DefLoc` is copied in multiple places across this file.
> 
> Unless there's a simple way to do it, I won't have time to make these changes in this PR.

Not necessarily. It's just that after looking at this again it seemed like comment is associated with a def and we have a structure that describes the def already, so it would be a more natural place for the doc string. I think it may be worth doing in the future, though.